src/Arcus.EventGrid.Tests.Unit/Artifacts/EventSamples.resx

```diff
@@ -167,16 +167,16 @@
       "version": 2,
       "properties": {
         "desired": {
-          "$metadata": {
-            "$lastUpdated": "2018-03-16T05:47:28.1359543Z"
```

Comment:
I removed the dolar sign ($) because otherwise these json files were not correctly parsed by the SDK types.
Strange I would say, since I guess these files comes directly from event grid testing?

Reply:
It's for a unit-test so ok, but those dollar signs have special meaning.  They indicate readonly elements in this case.